build-patched-kao-kcmo-images.sh

```diff
@@ -91,15 +91,18 @@ function patch_and_push_image() {
         rhpkg clone containers/crc-${image_name}
         pushd crc-${image_name}
         git remote add upstream https://pkgs.devel.redhat.com/git/containers/ose-${image_name}
-        # Just fetch the upstream/rhaos-${OCP_VERSION}-rhel-8 instead of all the branches and tags from upstream
-        git fetch upstream rhaos-${OCP_VERSION}-rhel-8 --no-tags
-        git checkout --track origin/rhaos-${OCP_VERSION}-rhel-8
-        git merge --no-ff -m "Merge commit ${vcs_ref} into rhaos-${OCP_VERSION}-rhel-8" -m "MaxFileSize: 104857600" ${vcs_ref}
+        # Just fetch the upstream/rhaos-${OCP_VERSION}-rhel-9 instead of all the branches and tags from upstream
+        git fetch upstream rhaos-${OCP_VERSION}-rhel-9 --no-tags
+        git checkout --track origin/rhaos-${OCP_VERSION}-rhel-9
+        git merge --no-ff -m "Merge commit ${vcs_ref} into rhaos-${OCP_VERSION}-rhel-9" -m "MaxFileSize: 104857600" ${vcs_ref}
         git push origin HEAD
-        rhpkg container-build --target crc-1-rhel-8-candidate
+        rhpkg container-build --target crc-1-rhel-9-candidate
```

Comment:
This change is not strictly related to what you describe in the commit log. Switching from rhaos-${OCP_VERSION}-rhel-8 to rhaos-${OCP_VERSION}-rhel-9 + the other changes in this commit achieve the switch from rhel8 to rhel9 base image in the Dockerfile 

Building in `crc-1-rhel-9-candidate` will run `podman build` in a rhel9 environment instead of a rhel8 environment, this should not make a significant difference in the built image, this will definitely not change the used base image.

Reply:
Right it is just change the build environment and not the image content, I just changed the occurrence of `8` to `9` , would be better to have some details in commit logs.

Reply:
In the cherry pick I updated the commit message.